app/presenters/avatar_presenter.rb

```diff
@@ -2,25 +2,25 @@
 # frozen_string_literal: true
 
 class AvatarPresenter < BasePresenter
-  DEFAULT_IMAGE = ActionController::Base.helpers.image_path("user/default.png")
+  def base_hash(with_default = false)
```

Comment:
Layout/SpaceAroundEqualsInParameterDefault: Surrounding space detected in default value assignment.